<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerSignalValuesBrowserWidgetForm</class>
 <widget class="QWidget" name="SeerSignalValuesBrowserWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="signalProfileComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Signal Profile Name.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>508</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="preferencesToolButton">
       <property name="toolTip">
        <string>Preferences</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/application-menu.svg</normaloff>:/seer/resources/RelaxLightIcons/application-menu.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="signalProfileLabel">
       <property name="text">
        <string>Profile</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QIndexTreeWidget" name="signalsTreeWidget">
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Stop</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Print</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pass</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QIndexTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">QIndexTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>signalProfileComboBox</tabstop>
  <tabstop>preferencesToolButton</tabstop>
  <tabstop>signalsTreeWidget</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
